ocean_provider/utils/datatoken.py

```diff
@@ -176,16 +184,16 @@ def verify_order_tx(
     except Exception as e:
         logger.error(e)
     logger.debug(f"Got events log when searching for OrderStarted : {event_logs}")
-    order_log = event_logs[0] if event_logs else None
+    order_log = None
```

Comment:
what about the order reused log? Shouldn't it also be handled similarly?

Reply:
no, because they are already filtered using data token address